Services/UI/OJS.Services.Ui.Business/Implementations/ParticipantsBusinessService.cs

```diff
@@ -53,8 +53,8 @@ public class ParticipantsBusinessService : IParticipantsBusinessService
         }
         else if (contest.IsOnsiteExam)
         {
-            participant.ParticipationStartTime = DateTime.Now;
-            participant.ParticipationEndTime = contest.EndTime;
+            participant.ParticipationStartTime = DateTime.UtcNow;
+            participant.ParticipationEndTime = contest.EndTime!.Value.ToUniversalTime();
```

Comment:
Isn't the `contest.EndTime` stored in UTC?

Reply:
And what about the null possibility, is it valid concern?

Reply:
I already added validation for it. I also added validation in the administration when saving or editing contest